# brain control interface message
Header header

int32 con_num
int32 con_key